charts/streams-app/README.md

```diff
@@ -0,0 +1,110 @@
+| `persistence.enabled`      | Use a [persistent volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) to store the state of the streams app. | `false` |
+| `persistence.size`         | Size of the persistent volume.                                                                                                    | `1Gi`   |
+| `persistence.storageClass` | Storage class to use for the persistent volume.                                                                                   |         |
+
+### Image
+
+| Parameter          | Description                                | Default      |
+|--------------------|--------------------------------------------|--------------|
+| `image`            | Docker image of the Kafka Streams app.     | `streamsApp` |
+| `imageTag`         | Docker image tag of the Kafka Streams app. | `latest`     |
+| `imagePullPolicy`  | Docker image pull policy.                  | `Always`     |
+| `imagePullSecrets` | Secrets to be used for private registries. |              |
+
+### Streams
+
+| Parameter                    | Description                                                                                                                                                                      | Default |
+|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `streams.brokers`            | Comma separated list of Kafka brokers to connect to.                                                                                                                             |         |
+| `streams.schemaRegistryUrl`  | URL of schema registry to connect to.                                                                                                                                            |         |
+| `streams.staticMembership`   | Whether to use [Kafka Static Group Membership](https://cwiki.apache.org/confluence/display/KAFKA/KIP-345%3A+Introduce+static+membership+protocol+to+reduce+consumer+rebalances). | `false` |
+| `streams.config`             | Configurations for your [Kafka Streams app](https://kafka.apache.org/documentation/#streamsconfigs).                                                                             | `{}`    |
+| `streams.inputTopics`        | List of input topics for your streams application.                                                                                                                               | `[]`    |
+| `streams.extraInputTopics`   | Map of additional named input topics.                                                                                                                                            | `{}`    |
```

Comment:
I think a context could help here: Used to name input topics for their usage, e.g. when joining different streams?